packages/concerto-core/test/data/parser/mapdeclaration/mapdeclaration.goodkey.scalar.boolean.cto

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace com.acme@1.0.0
+
+scalar BOOL extends Boolean
+
+map Dictionary {
+  o BOOL
```

Comment:
This one is interesting to me... It implies we are stringifying the boolean values "true" and "false" to use them as map keys - which sort of begs the question why not support stringifying numbers? We seem a bit inconsistent here.

Reply:
Based on our discussion, I've removed `Boolean` type from the legal key values.